tests/test_wps_split_merged_climos.py

```diff
@@ -40,54 +55,37 @@ def run_multiple_files(netcdfs):
 
 
 @pytest.mark.parametrize(
-    ("netcdf"), [(gcm_360_climos_local), (gcm_climos_local), (tasmax_climos_local),],
+    ("netcdf"), split_merged_climos_local,
 )
 def test_single_file_local(netcdf):
     run_single_file(netcdf)
```

Comment:
So, I believe testing `run_single_file` does not run with a list of files(multiple files). The variable `split_merged_climo_local` is obviously a list, but when it is parametrized `netcdf` argument is assigned an element of the list, and there are n-times assignments where n is the length of the list. In short, there will be n test cases for this particular test. It is clear when you run `pytest tests/test_wps_split_merged_climos.py::test_single_file_local`. pytest will show that there are 4 test cases, which is the number of local climo files in the test dataset. I think this confusion was probably made because of the brackets of `("netcdf")`.

Reply:
Ok that makes sense.  I didn't think `pytest` would automatically parametrize the list into individual cases.  Thanks for the explanation.